mod/wikirate/spec/set/right/contributed_analysis_spec.rb

```diff
@@ -5,11 +5,11 @@
 
     analysis = get_a_sample_analysis
 
-    article = Card[analysis.name].fetch :trait=>:wikirate_article,:new=>{}
+    article = Card[analysis.name].fetch :trait=>:overview,:new=>{}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.